src/scripts/createUser.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const readline = require('readline');
+const bcrypt = require('bcryptjs');
+const User = require('../server/models/User');
+
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+function askEmail() {
+	return new Promise((resolve, reject) => {
+		rl.question('Email: ', email => {
+			// See https://stackoverflow.com/a/46181/5116950
+			// eslint-disable-next-line
+			const regexEmail = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+			if (regexEmail.test(email))	resolve(email);
+			else reject(email);
+		});
+	});
+}
+
+function askPassword(email) {
+	return new Promise(resolve => {
+		rl.question('Password: ', password => { resolve([email, password]); });
+	});
+}
+
+function terminateReadline(message) {
+	if (message) console.log(message);
+	rl.close();
+	process.exit(0);
+}
+
+(async () => {
+	let emailResult;
+	try {
+		emailResult = await askEmail();
+	} catch (err) {
+		console.log('Invalid email, no user created');
+		rl.close();
+		process.exit(0);
+	}
+	const output = await askPassword(emailResult);
+	const email = output[0];
+	const password = output[1];
+
+	if (password.length < 8) {
+		terminateReadline('Weak password, no user created');
+	}
+	const hasUpperCase = /[A-Z]/.test(password);
+	const hasLowerCase = /[a-z]/.test(password);
+	const hasNumbers = /\d/.test(password);
+	const hasNonAlphas = /\W/.test(password);
+	if (hasUpperCase + hasLowerCase + hasNumbers + hasNonAlphas < 3) {
```

Comment:
Every single bit of modern research on the topic has found that these kinds of rules lead to weaker passwords. It's also insane to reject a password based on these rules without ever printing any kind of message saying that these rules exist. The clear solution here is to remove these rules entirely.